### ISSUE TITLE
Godot crashes and won't open, says "unable to initialize video driver"

### ISSUE THREAD

**Issue by Vitorgus** [godotengine/godot#67887]
### Godot version

3.5.1

### System information

Windws 11, GLES2, NVIDIA GeForce MX130, Driver GeForce Game Ready 522.25

### Issue description

After leaving the engine open for a while, it crashes. When trying to open it again the following warning appears: 

```
Unable to initialize Video driver

Your video card driver does not support any of the supported OpenGL versions.
Please update your drivers or if you have a very old or integrated GPU, upgrade it.
```

The only way to open the engine again, is to restart the computer. But it's only a matter of time until it crashes again, and I have to restart the computer again, and so the cycle goes.

I've tried to search online for the problem, but no solution worked for me.

Trying to run the terminal with the command `.\Godot_v3.5.1-stable_win64.exe -v -d -quiet` results in the following output

```
Godot Engine v3.5.1.stable.official.6fed1ffa3 - https://godotengine.org
ERROR: Error initializing GLAD
   at: is_viable (drivers/gles3/rasterizer_gles3.cpp:141)
```

### Steps to reproduce

Open the godot engine, leave it open for a while, and try to go back to it

**Comment by lawnjelly** [comment]
Did a previous version of Godot work on this PC, or with an earlier driver? Is this the first time you have tried Godot on this PC?

This sounds suspiciously like a driver bug such as a leak as a result of buffer renaming.

**Comment by Vitorgus** [comment]
I've recently upgraded some parts, did a fresh install and upgraded to windows 11. It's the first time I'm trying godot after that. But before, version 3.4 worked properly with windows 10 and with the older parts.

I've tried version 3.4.4, but it had the same issue, crashed after a while, with the same warning.

Versions 3.3.4 and 3.3.3 were a bit different. They would show the select project list, but would not open the project when clicking them. But wunning them through the CLI with the `-v -d -quiet` flags to check the logs... they would work normally. They would open the projects, works as expected, and didn't seemed to crash after a while.

Shouls I try with more versions of godot? Is there any other log or report I can generate to see what's causing this?

**Comment by godofgrunts** [comment]
It really sounds like something is crashing your GPU driver and it's falling back to your iGPU. I assume this is a laptop with an Intel cpu right?

I would check Event Viewer after the crash and see if you can find any errors or warning about the driver crashing.

You might also try forcing Windows to use the Nvidia card in the display settings or where ever that is. Maybe once it crashes it's trying to help and use the iGPU or something weird.

**Comment by Vitorgus** [comment]
Yes, it has an 8th gen Intel i7. Will try to check the event viewer, and changing the display settings, as suggested.

**Comment by clayjohn** [comment]
Another user recently had a similar issue and it turned out to be caused by their laptop dock. Unplugging it from the computer and plugging the monitor directly into the computer did the trick. 

The issue there was the dock was telling Windows that it had a GPU and for some reason that GPU was prioritized over both the iGPU and dedicated GPU in the computer.

**Comment by Vitorgus** [comment]
There was a recent GeForce Game Ready driver update to version 526.47 that seems to have fixed this. I've been testing it out for the past two days, to make sure it was the driver and not some other changes I made, and it seems to be the case. The problem hasn't appeared once since I updated.

Just wanted to thank everyone who took the time to reply and help try to diagnose the problem. I really had no idea what to do, or what could it be, and y'all shed some light on the problem.
